libtransmission/rpc-server.cc

```diff
@@ -832,60 +831,47 @@ void tr_rpc_server::set_anti_brute_force_enabled(bool enabled) noexcept
 
 // --- LIFECYCLE
 
-tr_rpc_server::tr_rpc_server(tr_session* session_in, tr_variant const& settings)
+tr_rpc_server::tr_rpc_server(tr_session* session_in, Settings settings)
```

Comment:
*(minor nit)* People with a text editor instead of an IDE might confuse `tr_rpc_server::Settings` with `libtransmission::Settings`. Maybe rename it to something like `RpcSettings`?

Reply:
...but I just use a text editor & not an  IDE? :thinking: 

I might change my mind on this, but I'm going to leave it like this for now.

Reply:
...hm, I kind of prefer the current naming, even though I use a text editor instead of an IDE.

I'll give it some thought, but for now I'm going to keep the current naming.

Reply:
Sure, fine by me.